datascience-notebook/Dockerfile

```diff
@@ -10,11 +10,25 @@ ENV CHOWN_HOME=no
 
 COPY ./files/apt_packages /tmp/
 
-RUN conda install boto3 black flake8 jupyterlab_code_formatter \
+RUN mkdir /opt/oracle \
```

Comment:
I'd add a comment above this command to explain what's installing, etc...some clues for the future us (or them).